### PR TITLE
Replicate MS MARCO Passgae/Document Retrieval with Pyserini

### PR DESCRIPTION
**Pull request by larryli1999**
# Environment
- Python 3.6.1
- Google Colab

# Issue
Experiencing difficulties to use trec_eval to compute the metrics in the passage retrieval experiment due to the following issue:
trec_eval.get_qrels: Cannot read qrels file 'collections/msmarco-passage/qrels.dev.small.trec'
trec_eval: Quit in file 'collections/msmarco-passage/qrels.dev.small.trec'

Other results are the same as the given results from the documentation.

# My results
passage
Capture3

document
Capture4

### CODE REVIEW

**Comment by larryli1999** [comment]
trec_eval.get_qrels: Cannot read qrels file 'collections/msmarco-passage/qrels.dev.small.trec'
trec_eval: Quit in file 'collections/msmarco-passage/qrels.dev.small.trec'

Other results are the same as the given results from the documentation

# My results
passage
Capture3
document
Capture4

**Comment by lintool** [comment]
Let's try to sort out the qrels issue - I believe the relevant fragment from the guide is here:

Screen Shot 2021-01-22 at 5 06 35 PM

Do you have an idea of what didn't work? Can you help with some basic debugging?

**Comment by lintool** [comment]
Sorry, I just noticed, this is pyserini - above snippet was copied from anserini guide: https://github.com/castorini/anserini/blob/master/docs/experiments-msmarco-passage.md

Are we missing the command here?

**Comment by lintool** [comment]
Compare with https://github.com/castorini/pyserini/pull/332 ?

**Comment by larryli1999** [comment]
Hi Professor Lin. I noticed that this part is a bit different between the previous replicated results and the documentation:

**Comment by lintool** [comment]
yes, I recently updated the documentation and might have broken something... can you help me go through and fix what's broken?

**Comment by lintool** [comment]
@larryli1999 Can you please update? https://github.com/castorini/pyserini/commit/0d6595aef9abed12901308d5ba2239fa233731b0 doesn't seem to be pointing to a valid commit on master branch.

File: docs/experiments-msmarco-doc.md [hunk 1]
```diff
@@ -134,3 +134,4 @@ We can see that Anserini's (tuned) BM25 baseline is already much better than the
 + Results replicated by [@HEC2018](https://github.com/HEC2018) on 2021-01-04 (commit [`46a6d47`](https://github.com/castorini/pyserini/commit/46a6d472267a559152495d004c2a12f8e95e53f0))
 + Results replicated by [@KaiSun314](https://github.com/KaiSun314) on 2021-01-08 (commit [`aeec31f`](https://github.com/castorini/pyserini/commit/aeec31fbe17d39ecf3081597b4832f5af57ea549))
 + Results replicated by [@yemiliey](https://github.com/yemiliey) on 2021-01-18 (commit [`98f3236`](https://github.com/castorini/pyserini/commit/98f323659c8a0a5d8ef26bb3f6768458a34e3eb9))
++ Results replicated by [@larryli1999](https://github.com/larryli1999) on 2021-01-22 (commit [`0d6595a`](https://github.com/castorini/pyserini/commit/0d6595aef9abed12901308d5ba2239fa233731b0))
```
Comment (lintool) [inline docs/experiments-msmarco-doc.md:137]: hi @larryli1999 if I go to this URL, it says "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

Comment (larryli1999) [reply docs/experiments-msmarco-doc.md:137]: Ok, let me double check.

**Comment by lintool** [comment]
I'm still having same issues with "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

Something I'm doing wrong?

**Comment by larryli1999** [comment]
Sorry for the messy commits, can you please check again? I think it should be fine this time.

**Review by lintool — APPROVED** [review]

**Pull request merged**